From: Michal Hocko <mhocko@suse.com>
Subject: kabi: x86/speculation/l1tf: Increase l1tf memory limit for Nehalem+
Patch-mainline: no, suse specific
References: bnc#1105536

hide new x86_cache_bits from the kabi checker. Nobody should be touching
this part of the structure. There is already a kabi fixup so put the new
addition into the same one.

Signed-off-by: Michal Hocko <mhocko@suse.com>

---
 arch/x86/include/asm/processor.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/x86/include/asm/processor.h
+++ b/arch/x86/include/asm/processor.h
@@ -131,10 +131,10 @@ struct cpuinfo_x86 {
 	/* Index into per_cpu list: */
 	u16			cpu_index;
 	u32			microcode;
-	/* Address space bits used by the cache internally */
-	u8			x86_cache_bits;
 #ifndef __GENKSYMS__
 	unsigned		initialized : 1;
+	/* Address space bits used by the cache internally */
+	u8			x86_cache_bits;
 #endif
 };
 
